db/migrate/20160414094300_change_capacity_to_hash_from_persistent_volume.rb

```diff
@@ -0,0 +1,41 @@
+class ChangeCapacityToHashFromPersistentVolume < ActiveRecord::Migration[5.0]
+  class ContainerVolume < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class PersistentVolume < ContainerVolume
+    self.inheritance_column = :_type_disabled # disable STI
+    serialize :capacity
+  end
+
+  def up
+    change_column :container_volumes, :capacity, :text
+    say_with_time("Changing string to hash") do
+      PersistentVolume.all.each do |vol|
```

Comment:
not sure how many PersistentVolumes might be there, but I would always try to avoid `.all` and go with `.find_each`